src/menu/views/settings_editor.c

```
@@ -359,98 +425,221 @@ static void draw (menu_t *menu, surface_t *d) {
+                    total_length += strlen(row_texts[rendered_row_count]);
+                }
+                rendered_row_count++;
+            }
+            break;
+        }
+    }
+
+    if (rendered_row_count > 0) {
+        rdpq_paragraph_t *layout;
+        rdpq_paragraph_t *settings_list_layout = malloc(sizeof(rdpq_paragraph_t) + (sizeof(rdpq_paragraph_char_t) * total_length));
+        memset(settings_list_layout, 0, sizeof(rdpq_paragraph_t));
```

Cppcheck (2.21.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)